### PR TITLE
GTC-2620 Palm risk 2023 update

### PR DESCRIPTION
**Pull request by manukala6**
## Pull request checklist

Please check if your PR fulfills the following requirements:
- [ ] Make sure you are requesting to pull a topic/feature/bugfix branch (right side). Don't request your master!
- [ ] Make sure you are making a pull request against the develop branch (left side). Also you should start your branch off our develop.
- [ ] Check the commit's or even all commits' message styles matches our requested structure.
- [ ] Check your code additions will fail neither code linting checks nor unit test.

## Pull request type

 

 

Please check the type of change your PR introduces:
- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe): 

## What is the current behavior?

Max year for Palm risk tool is 2021

Issue Number: [GTC-2620](https://gfw.atlassian.net/browse/GTC-2620)

## What is the new behavior?

- Max year for Palm risk tool is 2022
- Peatlands updated to latest (v20200807 -> v20230315)
- SDPT updated to latest (v20201209 -> v20230911)
- WDPA non-commercial updated to latest (v202208 -> v202210)
 
## Does this introduce a breaking change?

- [ ] Yes
- [x] No

## Other information

[GTC-2620]: https://gfw.atlassian.net/browse/GTC-2620?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/main/resources/raster-catalog-pro.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
     },
     {
       "name":"wdpa_protected_areas",
-      "source_uri":"s3://gfw-data-lake/wdpa_protected_areas/v202208/raster/epsg-4326/{grid_size}/{row_count}/iucn_cat/gdal-geotiff/{tile_id}.tif"
+      "source_uri":"s3://gfw-data-lake/wdpa_protected_areas/v202210/raster/epsg-4326/{grid_size}/{row_count}/iucn_cat/gdal-geotiff/{tile_id}.tif"
```
Comment (danscales) [inline src/main/resources/raster-catalog-pro.json:13]: Did you mean to make this change?  s3://gfw-data-lake/wdpa_protected_areas/v202210 doesn't seem to exist.  Also, FCD doesn't use this dataset anymore (it uses detailed_wda_protected_areas, which uses the licensed WDPA dataset).

File: src/main/scala/org/globalforestwatch/summarystats/forest_change_diagnostic/ForestChangeDiagnosticAnalysis.scala [hunk 2]
```diff
@@ -220,7 +220,7 @@ object ForestChangeDiagnosticAnalysis extends SummaryAnalysis {
       }
       .reduceByKey(_ merge _)
       .mapValues { fires =>
-        aggregateFireData(fires.merge(ForestChangeDiagnosticDataLossYearly.prefilled)).limitToMaxYear(2021)
+        aggregateFireData(fires.merge(ForestChangeDiagnosticDataLossYearly.prefilled)).limitToMaxYear(2022)
```
Comment (danscales) [inline src/main/scala/org/globalforestwatch/summarystats/forest_change_diagnostic/ForestChangeDiagnosticAnalysis.scala:223]: Can you update the 2021 comment above at line 204?

**Review by danscales — COMMENTED** [review]
Looks good except for my two comments.

**Review by jterry64 — APPROVED** [review]
Everything looks good, except the comment @danscales made about the WDPA version!

**Comment by manukala6** [comment]
@jterry64 @danscales Updated WDPA to the correct version and the comment with the wrong year

**Review by danscales — APPROVED** [review]

**Pull request merged**